composer.json

```diff
@@ -62,12 +62,22 @@
     "autoload": {
         "psr-4": {
             "EzSystems\\PlatformInstallerBundle\\": "eZ/Bundle/PlatformInstallerBundle/src",
-            "EzSystems\\PlatformBehatBundle\\": "eZ/Bundle/PlatformBehatBundle"
+            "EzSystems\\PlatformBehatBundle\\": "eZ/Bundle/PlatformBehatBundle",
+            "Ibexa\\Bundle\\Core\\": "src/bundle",
```

Comment:
`src/bundle/Core`? Because this is multibundle repo right?

Reply:
What does the current rebranding mapping say about those? Every bundle in a separate namespace?

Reply:
> What does the current rebranding mapping say about those? Every bundle in a separate namespace?

@Steveb-p [yes](https://github.com/ibexa/namespace-compatibility/blob/main/src/bundle/Resources/mappings/input-maps/ezsystems/ezplatform-kernel.json#L2740-L2781)

Reply:
Fixed in 3b5844fe.

Each bundle has it's own separate namespace declaration, because `Ibexa\Bundle` prefix is too generic and would cause composer autoloader to look into kernel package for nearly every Ibexa bundle.